packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -1096,6 +1096,11 @@ class AdbLogReader extends DeviceLogReader {
     RegExp(r'^[F]\/[\S^:]+:\s+'),
   ];
 
+  // E/SurfaceSyncer(22636): Failed to find sync for id=9
+  // Some versions of Android spew this out. It is inactionable to the end user
+  // and causes no problems for the application.
```

Comment:
If you have a link to the documentation you used to figure this out including it here could help future maintainers.

Reply:
I linked it in the bug linked in this PR - unfortunately it's not visible to non-googlers, but perhaps I should have included it here too...